package.json

```diff
@@ -8,24 +8,25 @@
     "eject": "expo eject"
   },
   "dependencies": {
+    "@react-native-community/masked-view": "0.1.10",
     "@react-navigation/drawer": "^5.11.5",
     "@react-navigation/native": "^5.9.0",
     "expo": "~40.0.0",
     "expo-status-bar": "~1.0.3",
     "native-base": "^3.0.0-next.11",
+    "prop-types": "^15.7.2",
```

Comment:
I don't think this is needed.

Reply:
PropType is needed as I was passing props to other component.

Reply:
No, it is not. Read about it.